src/main.ts

```diff
@@ -0,0 +1,8 @@
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from './app.module';
+
+async function bootstrap() {
+  const app = await NestFactory.create(AppModule);
```

Comment:
We need app config such cors, express.json(), helmet

After  `const app = await NestFactory.create(AppModule);`
add these:
app.use(cors())
app.use(express.json())
app.use(helmet())